utils/extractor/src/index.ts

```diff
@@ -80,7 +82,19 @@ export const Extractor = (
             await asyncBatch(
               capture[sheet.name].data,
               async (chunk) => {
-                await api.records.insert(sheet.id, chunk)
+                const body = zlib.gzipSync(JSON.stringify(chunk))
+                await axios.post(
```

Comment:
Why are we using a direct `axios` call instead of the `@flatfile/api` here?

Reply:
The Fern API doesn't support gzipped payloads